bridge/src/main/java/com/bitwarden/bridge/util/EncryptionUtils.kt

```diff
@@ -0,0 +1,201 @@
+package com.bitwarden.bridge.util
```

Comment:
S/O to @brian-livefront for originally writing all this code in his POC. I'm standing on the shoulders of a giant!